translate/storage/test_po.py

```diff
@@ -322,6 +322,17 @@ def test_nongettext_location(self):
         assert locations[0] == u"programming/C/programming.xml:44(para)"
         assert isinstance(locations[0], six.text_type)
 
+    def test_percent_location(self):
+        """test that we correctly handle a location with percent chars"""
+        posource = '#: /foo/bar/%%var%%www%%about.html:44\nmsgid "test"\nmsgstr "rest"\n'
```

Comment:
Next times please use a multiline string since it makes easier to copy paste actual examples without need to escape chars. Also it is easier to read by humans.